content/docs/command-reference/list-url.md

```diff
@@ -0,0 +1,97 @@
+# list-url
+
+> Aliased to `dvc ls-url`
+
+List contents from a supported URL (for example `s3://`, `ssh://`, and other
+protocols).
+
+> Useful to find data to `dvc get-url` or `dvc import-url`.
+
+## Synopsis
+
+```usage
+usage: dvc ls-url [-h] [-q | -v] [-R] url
+
+positional arguments:
+  url              See `dvc get-url` for full list of supported URLs.
+```
+
+## Description
+
+Lists files and directories from an external location. `dvc ls-url` is
+equivalent to a wrapper providing a uniform interface around various listing
+commands such as `aws s3 ls` or `ssh user@host ls -a`.
+
+The `url` argument specifies the location of the data to be listed. It supports
+several kinds of external data sources:
+
+| Type    | Description                  | `url` format example                  |
+| ------- | ---------------------------- | ------------------------------------- |
+| `s3`    | Amazon S3                    | `s3://bucket/data`                    |
+| `azure` | Microsoft Azure Blob Storage | `azure://container/data`              |
+| `gs`    | Google Cloud Storage         | `gs://bucket/data`                    |
+| `ssh`   | SSH server                   | `ssh://user@example.com/path/to/data` |
+| `local` | Local path                   | `/path/to/local/data`                 |
+
+> If you installed DVC via `pip` and plan to access cloud services as external
+> data sources, you might need to install these optional dependencies: `[s3]`,
+> `[azure]`, `[gs]`, `[oss]`, `[ssh]`. Alternatively, use `[all]` to include
+> them all. The command should look like this: `pip install "dvc[s3]"`. (This
+> example installs `boto3` library along with DVC to support S3 storage.)
+
+Only the root directory is listed by default, but the `-R` option can be used to
+list files recursively.
+
+## Options
+
+- `-R`, `--recursive` - recursively list files of all subdirectories.
```

Comment:
Minor: Maybe use the same wording as in https://dvc.org/doc/command-reference/list#options ? I.e.

```suggestion
- `-R`, `--recursive` - recursively prints contents of all subdirectories.
```

> Or perhaps update the `list` ref for language consistency.

Reply:
I want to emphasise that the command only returns files (not directories). I updated the `list` ref which wrongly implied that `dvc ls -R` returns directories.